migrations/sql/V2020.07.15.08.33__update_bond_view_with_closed_date_and_notes.sql

```diff
@@ -0,0 +1,79 @@
+DROP VIEW IF EXISTS bond_view;
+CREATE VIEW bond_view AS
```

Comment:
What is bond_view used for? In general we have chosen to avoid views unless there is a constraint or performance concern that makes them a significantly better choice than working in our ORM

Reply:
The bond view was created to support securities reporting, to almagate the bond and bond_history tables for easier querying. If you take a look at the securities dashboard reports you can see how it's used.
https://github.com/bcgov/mds/pull/1359

Reply:
sounds good